library/percona

```diff
@@ -32,7 +32,7 @@ GitCommit: 3a2779e6937f832e957980cc3847f5efa7658ea9
 Directory: percona-server-mongodb.40
 File: Dockerfile
 
-Tags: psmdb-3.6.18, psmdb-3.6
-GitCommit: 3a2779e6937f832e957980cc3847f5efa7658ea9
+Tags: psmdb-3.6.19, psmdb-3.6
+GitCommit: ae69f3156098b2345492e5d2038e56c4a8e2ee0d
 Directory: percona-server-mongodb.36
```

Comment:
It looks like this directory needs to be `percona-server-mongodb-3.6` now because of https://github.com/percona/percona-docker/pull/277.

Reply:
Thanks, forgot about folders names change.